mediacloud/mediawords/util/test_colors.py

```diff
@@ -0,0 +1,86 @@
+from mediawords.test.test_database import TestDatabaseWithSchemaTestCase
+from mediawords.util.colors import *
+
+
+def test_hex_to_rgb():
+    assert hex_to_rgb('ff0000') == (255, 0, 0,)
+    assert hex_to_rgb('FFFFFF') == (255, 255, 255,)
+    assert hex_to_rgb('#ff0000') == (255, 0, 0,)
+    assert hex_to_rgb('#FFFFFF') == (255, 255, 255,)
+
+
+def test_rgb_to_hex():
+    assert rgb_to_hex(255, 0, 0).lower() == 'ff0000'
+    assert rgb_to_hex(0, 0, 0).lower() == '000000'
+    assert rgb_to_hex(255, 255, 255).lower() == 'ffffff'
+
+
+def test_analogous_color():
+    starting_color = '0000ff'
+    colors = analogous_color(color=starting_color, my_slices=255, slices=255)
+    assert len(colors) == 256
```

Comment:
the docstring seems to suggest this should be 255, or whatever value `my_slices` has

Reply:
Sorry, mindlessly copied [`Color::Mix`'s comment](https://metacpan.org/pod/Color::Mix#analogous()). Updated docstring in an attempt to clarify things a little.